examples/demo-vendoring/nested-vendor/vendor1.yaml

```
@@ -0,0 +1,14 @@
+metadata:
+  name: vendor-test-1
+  description: Test vendor config 1
+spec:
+  sources:
+    - component: "ipinfo"
+      source: "github.com/cloudposse/atmos.git//examples/demo-library/{{ .Component }}?ref={{.Version}}"
+      version: "main"
+      targets:
+        - "components/terraform/{{ .Component }}/{{.Version}}"
+      tags:
+        - demo
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)